packages/onramp-kit/example/client/README.md

```diff
@@ -12,3 +12,16 @@ This example can be used as a playground to test the package as it is relatively
 ```typescript
 import { SafeOnRampKit, SafeOnRampProviderType } from '../../src/index'
 ```
+
+To use the example properly in your local machine follow these steps:
+
+**In the project root**
+
+1. `yarn install`
+2. `yarn build`
+
+**In the onramp-kit client example root**
```

Comment:
In case it is edited above, it would apply also here
```suggestion
**In the project root folder**

1. `yarn install`
2. `yarn build`

**In the onramp-kit client example root folder**
```